test/all.test.js

```diff
@@ -527,6 +558,9 @@ describe(Support.getTestDialectTeaser('Tests'), function () {
 					})
 					.then(function() {
 						return this.drive.findAll({
+							where: {
+								name: 'x'
+							},
```

Comment:
Limit the loaded drives to the one the test was intended for. As all folders are put under a drive now the result of the original call changed